isign/signable.py

```diff
@@ -95,6 +95,10 @@ def _get_arch(self, macho, arch_offset, arch_size):
             macho.ncmds -= 1
             macho.commands = macho.commands[:-1]
 
+            # We're stripping out the fake LC_CODE_SIGNATURE command, which we know has a size of 16, so we need to
```

Comment:
Move this up with the other two things being decremented and put the comment above all three? While it's one of our few changes to this code, it belongs right next to the other parts.